app/indexers/administrative_tag_indexer.rb

```diff
@@ -15,15 +15,20 @@ def initialize(id:, administrative_tags:, **)
   end
 
   # @return [Hash] the partial solr document for administrative tags
-  def to_solr
+  def to_solr # rubocop:disable Metrics/MethodLength
     Rails.logger.debug { "In #{self.class}" }
 
-    solr_doc = { 'tag_ssim' => [], 'exploded_nonproject_tag_ssim' => [] }
+    solr_doc = {
+      'tag_ssim' => [], # deprecated in favor of tag_text_unstemmed_sim for searcahbility
```

Comment:
Teensy typo in the comment here: `searcahbility`

Reply:
well spotted, thank you!   corrected.